pkg/virt-launcher/virtwrap/converter/converter.go

```diff
@@ -1482,108 +1457,95 @@ func Convert_v1_VirtualMachine_To_api_Domain(vmi *v1.VirtualMachineInstance, dom
 
 	networks := indexNetworksByName(vmi.Spec.Networks)
 
-	sriovPciAddresses := make(map[string][]string)
-	for key, value := range c.SRIOVDevices {
-		sriovPciAddresses[key] = append([]string{}, value...)
-	}
-
 	for i, iface := range vmi.Spec.Domain.Devices.Interfaces {
 		net, isExist := networks[iface.Name]
 		if !isExist {
 			return fmt.Errorf("failed to find network %s", iface.Name)
 		}
 
 		if iface.SRIOV != nil {
-			var pciAddr string
-			pciAddr, sriovPciAddresses, err = popSRIOVPCIAddress(iface.Name, sriovPciAddresses)
-			if err != nil {
-				return fmt.Errorf("failed to configure SRIOV %s: %v", iface.Name, err)
-			}
-			hostDev, err := createSRIOVHostDevice(pciAddr, iface.PciAddress, iface.BootOrder)
+			continue
+		}
+
+		ifaceType := getInterfaceType(&vmi.Spec.Domain.Devices.Interfaces[i])
+		domainIface := api.Interface{
+			Model: &api.Model{
+				Type: translateModel(c, ifaceType),
+			},
+			Alias: &api.Alias{
+				Name: iface.Name,
+			},
+		}
+
+		// if UseEmulation unset and at least one NIC model is virtio,
+		// /dev/vhost-net must be present as we should have asked for it.
+		var virtioNetMQRequested bool
+		if mq := vmi.Spec.Domain.Devices.NetworkInterfaceMultiQueue; mq != nil {
+			virtioNetMQRequested = *mq
+		}
+		if ifaceType == "virtio" && virtioNetProhibited {
+			return fmt.Errorf("In-kernel virtio-net device emulation '/dev/vhost-net' not present")
+		} else if ifaceType == "virtio" && virtioNetMQRequested {
+			queueCount := uint(CalculateNetworkQueues(vmi))
+			domainIface.Driver = &api.InterfaceDriver{Name: "vhost", Queues: &queueCount}
+		}
+
+		// Add a pciAddress if specified
+		if iface.PciAddress != "" {
```

Comment:
Super noob question, do we have PciAddress in case of non SRIOV iface ? if this is the case we are not going to reach this condition.